web/annotation_layer_builder.js

```diff
@@ -13,6 +13,8 @@
  * limitations under the License.
  */
 
+// eslint-disable-next-line max-len
+/** @typedef {import("./interfaces").IPDFAnnotationLayerFactory} IPDFAnnotationLayerFactory */
```

Comment:
Here, and elsewhere, you really ought to put a blank line between this sort of comment and the imports below since as-is this is *somewhat* hurting readability.